test/transmission/test.py

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/python
```

Comment:
Was this file unused before? If so, may I ask you to to this change in a separate PR?

Reply:
Done.